docs/docs/reference/built-in-components/gatsby-slice.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Gatsby Slice API
```

Comment:
Is it "Gatsby Slice" API or "Gatsby Slice**s**" API?

Reply:
I was going off the naming pattern of other components. I know our RFC calls it "Slices", but I think matching the others (i.e. Gatsby Script API) is the best path forward.